CHANGELOG.md

```diff
@@ -25,6 +25,13 @@ The format is loosely based on [Keep a Changelog](http://keepachangelog.com/en/1
 
 * Added support for setup and verification of the event handlers through `Setup[Add|Remove]` and `Verify[Add|Remove|All]` (@lepijohnny, #825) 
 
+* Support for matching generic type arguments: `mock.Setup(m => m.Method<It.IsAnyType>(...))`. Create your own type matchers by defining a type that implements `ITypeMatcher`: `class ExceptionsOnly : ITypeMatcher { public bool Matches(Type type) => typeof(Exception).IsAssignableFrom(type); }`. If your type matcher needs to be a delegate or enum, decorate it with a `[TypeMatcher]` custom attribute. (@stakx, #908)
+
+* New overloads for `setup.Callback` and `setup.Returns` where the callback receives the current `IInvocation`:  
+   - `setup.Callback(new InvocationAction(invocation => ...))`
+   - `setup.Returns(new InvocationFunc(invocation => ...))`
+   These are indended to be used in scenarios where type matchers are used as a `IInvocation` allows discovery of the used type arguments (besides the regular arguments). (@stakx, #908)
```

Comment:
* missing line break between this line and the preceding one
* typo: `indended` &rarr; `intended`